### PR TITLE
Fix removes duplicate primitive mods

### LINKED ISSUE #192: Duplicate Primitives crates
There are two primitives crate, node-primitives and primitives. Ideally, we don't want this, move all logic inside node-primitives to primitives crate and deduplicate

### PR DESCRIPTION
**Pull request by mambisi**
Resolves #192

### CODE REVIEW

File: node/Cargo.toml [hunk 1]
```diff
@@ -82,7 +82,7 @@ frame-system = { version = "3.0.0", default-features = false, git = "https://git
 pallet-transaction-payment = { default-features = false, version = '3.0.0', git = "https://github.com/paritytech/substrate" }
 
 #Primitives
-node-primitives = { path = "../node-primitives" }
+polkadex-primitives = { git = "https://github.com/Polkadex-Substrate/Polkadex.git", branch = 'develop', default-features = false }
```
Comment (Gauthamastro) [inline node/Cargo.toml:85]: @mambisi Please use a local path here

File: pallets/fungible-assets/Cargo.toml [hunk 2]
```diff
@@ -18,7 +18,7 @@ sp-io = {  git = "https://github.com/paritytech/substrate", default-features = f
 sp-core = {  git = "https://github.com/paritytech/substrate", default-features = false }
 sp-std = {  git = "https://github.com/paritytech/substrate", default-features = false }
 sp-runtime = {  git = "https://github.com/paritytech/substrate", default-features = false }
-polkadex-primitives = { path = "../../primitives", default-features = false }
+polkadex-primitives = { git = "https://github.com/Polkadex-Substrate/Polkadex.git", branch = 'develop', default-features = false }
```
Comment (Gauthamastro) [inline pallets/fungible-assets/Cargo.toml:21]: ^^^^

File: primitives/src/lib.rs [hunk 3]
```diff
@@ -16,7 +16,106 @@
...
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
+
+// This file is part of Substrate.
+
+// Copyright (C) 2018-2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
```
Comment (Gauthamastro) [inline primitives/src/lib.rs:26]: Why is Parity's License here?

**Review by Gauthamastro — CHANGES_REQUESTED** [review]

**Comment by Gauthamastro** [comment]
CI is failing too

File: node/Cargo.toml (hunk 1 above)
Comment (mambisi) [reply node/Cargo.toml]: fixed in latest change

File: pallets/fungible-assets/Cargo.toml (hunk 2 above)
Comment (mambisi) [reply pallets/fungible-assets/Cargo.toml]: Ok

File: primitives/src/lib.rs (hunk 3 above)
Comment (mambisi) [reply primitives/src/lib.rs]: ok removed

**Comment by ksr30** [comment]
@Gauthamastro can we merge it?

**Review by Gauthamastro — APPROVED** [review]

**Pull request merged**